tests/test_builders/test_build_linkcheck.py

```diff
@@ -1448,6 +1448,12 @@ class CaseSensitiveHandler(BaseHTTPRequestHandler):
 
     def do_HEAD(self):
         if self.path == '/path':
+            # Redirect lowercase /path to uppercase /Path
+            self.send_response(301, 'Moved Permanently')
+            self.send_header('Location', f'http://{self.headers["Host"]}/Path')
```

Comment:
It's OK for the `Location` to refer to a relative path, so this can be simplified:

```suggestion
            self.send_header('Location', '/Path')
```